apps/storybook/stories/FeaturedQuoteBlock.stories.tsx

```diff
@@ -2,14 +2,12 @@ import type { Meta, StoryObj } from '@storybook/react';
 import FeaturedQuoteBlock from '../../next-evans/components/pages/friend/FeaturedQuoteBlock';
 import '../styles/globals.css';
 
-const meta: Meta<typeof FeaturedQuoteBlock> = {
-  title: 'Friend/FeaturedQuoteBlock',
+const meta = {
+  title: `Friend/FeaturedQuoteBlock`,
   component: FeaturedQuoteBlock,
-};
-
-export default meta;
+} satisfies Meta<typeof FeaturedQuoteBlock>;
 
-type Story = StoryObj<typeof FeaturedQuoteBlock>;
+type Story = StoryObj<typeof meta>;
```

Comment:
this change, and the change on line 8 above (`satisfies...`) is what allows us to get _strict_ typing for our storybook story props.  it's new in storybook 7.